lib/smart_answer/calculators/uk_benefits_abroad_calculator.rb

```diff
@@ -106,6 +106,10 @@ def employer_paying_ni_not_ssp_country_entitled?
       (COUNTRIES_OF_FORMER_YUGOSLAVIA + %w[barbados guernsey jersey israel turkey]).include?(country)
     end
 
+    def jersey_guernsey?
```

Comment:
Minor: This might be better renamed as `channel_islands?`? (Would need to rename the outcome too). It's better to give the name of the group of countries if possible, just like we do with EEA countries.